conans/client/tools/files.py

```diff
@@ -63,8 +61,15 @@ def unzip(filename, destination=".", keep_permissions=False, pattern=None):
     that the zip was created correctly.
     :param pattern: Extract only paths matching the pattern. This should be a
     Unix shell-style wildcard, see fnmatch documentation for more details.
+    :param output: output
     :return:
     """
+    if output is None:
```

Comment:
This is highly repeated, move to function?

Reply:
It is to be removed, I prefer to have it repeated and delete it one by one than writing a function that may be used by someone else and pervive, or may become dead code. But no strong opinion about this.